css/index.css

```diff
@@ -112,6 +142,46 @@ p {
     justify-content: center;
 }
 
+.middle-content .boxes :nth-child(1) {
+    background-color: teal;
```

Comment:
I like the psuedo selectors for targeting, don't forget that for first and last child they can be written as such `<target> :firstchild` and last. again quality of life so that you and other devs know where you are starting and ending.